packages/drivers/odsp-driver/src/odspCache.ts

```diff
@@ -271,3 +295,28 @@ export function createOdspCache(
         persistedCache: new PersistedCacheWithErrorHandling(persistedCache, logger),
     };
 }
+
+export interface IPersistedCacheValueWithEpoch {
+    value: any,
+    fluidEpoch: string | undefined,
+}
+
+export class LocalPersistentCacheAdapter implements IPersistedCache {
+    constructor(private readonly cache: IPersistedCache) {}
+
+    async get(entry: ICacheEntry, expiry?: number): Promise<IPersistedCacheValueWithEpoch> {
+        return this.cache.get(entry, expiry) as Promise<IPersistedCacheValueWithEpoch>;
```

Comment:
I like how it's organized, but I'll point out that while API is the same, we are still not backward compatible, because old values in storage right now are not in the format we expect them here. 
So we either need to add note to breaking.md to clear cache and change its schema, or we need to have detection of old format and convert it as not having epoch.

If SPO already provides epoch on all APIs, then we can go with a solution that simply deleted old cached entries if they are in old format / have no epoch.

If SPO does not return yet epoch on all APIs, then we can just return non-versioned cache (just convert to different format), but in the future we would need to clear all those non-epoched versions from cache

Reply:
So spo does not return epoch on all apis yet. I was thinking on converting values with no epoch to with epoch inside cache but feels that it is hard to check that whether a value is "IPersistedCacheValueWithEpoch" because epoch can be undefined in this interface. So I have added a note in breaking,md to let host know that schema has chanegs for entries and they need to clear the cache in this version.

Reply:
Ok, so we need to plan another bump in cache version (on host side) when epoch happens on all APIs